CONTRIBUTING.md

```diff
@@ -1,34 +1,45 @@
-# Contributing
+# Contributing to the project
 
 Oracle welcomes contributions to this repository from anyone.
 
 If you want to submit a pull request to fix a bug or enhance an existing feature, please first open an issue and link to that issue when you submit your pull request.
 
 If you have any questions about a possible submission, feel free to open an issue too.
 
-## Contributing to the repository
+## Contributing code
 
-Pull requests can be made under [The Oracle Contributor Agreement](https://oca.opensource.oracle.com/).
+Before submitting code via a pull request, you will need to have signed the [Oracle Contributor Agreement][OCA] (OCA) and your commits need to include the following line using the name and e-mail address you used to sign the OCA:
```

Comment:
"Will need to have signed" is passive, replace with "must sign"
replace "need to" with must - be specific (also removes passive voice).